.meteor/packages

```diff
@@ -74,3 +74,7 @@ perak:markdown
 seriousm:emoji-continued
 templates:tabs
 verron:autosize
+
+http
+ejson
+d4nyll:epic
```

Comment:
You should move `http` and `ejson` to the `Utilities` and `epic` to the UI components. Packages are sorted alphabetically — with official packages first.